include/libassert/assert.hpp

```diff
@@ -137,6 +137,12 @@
  #pragma warning(disable: 4251; disable: 4275)
 #endif
 
+#if __cplusplus >= 2020002
+ #define LIBASSERT_VA_ARGS(...) __VA_OPT__(,) __VA_ARGS__
+#else
```

Comment:
Can you update this to `LIBASSERT_IS_GCC || __cplusplus >= 2020002`. The library still needs to workaround https://gcc.gnu.org/bugzilla/show_bug.cgi?id=44317 (and if you could re-add the comment that'd be great).

Reply:
sure